content/articles/domain-apex-heroku.markdown

```diff
@@ -16,19 +16,19 @@ categories:
 
 Your domain apex is the second or third level domain provided to you by your domain registrar. For example, `dnsimple.com` is the apex domain for DNSimple.
 
-To point your apex to Heroku you have 2 options:
+To point your apex to Heroku, you have 2 options:
 
 1. Use the Heroku one-click service.
 1. Manually add an [ALIAS record](/articles/alias-record).
 
 
-Once your apex domain is pointing to correct place you will have to configure the custom domain at Heroku.
+Once your apex domain is pointing to correct place, you will have to configure the custom domain at Heroku.
 
 ## Using the Heroku one-click service
 
-You will have to choose what one-click service to apply depending on whether your app uses SSL or not. If your app does use SSL you will have to use the `Heroku SSL` service. If your app does not use SSL you will have to go for the one named just `Heroku`.
+Our one-click service for Heroku is listed under the "Infrastructure" tab:
 
-![Heroku services](/files/heroku-services.jpg)
```

Comment:
Should https://github.com/dnsimple/dnsimple-support/blob/master/content/files/heroku-services.jpg be deleted?

Reply:
I thought I had deleted it 🤔 I can fix that up because it probably should be.

Reply:
Deleted and new PR submitted